v2/pkg/contextargs/contextargs.go

```diff
@@ -0,0 +1,19 @@
+package contextargs
+
+import (
+	"net/http/cookiejar"
+	"sync"
+)
+
+type Context struct {
+	Input     string
```

Comment:
We should have input and other arguments as unexported and a method to return these values since that way the interface would be well defined. Maybe Getter and Setter type pattern would be better since that way we can easily track changes / change interface implementation? just thinking out loud

Reply:
I changed the implementation to helper getter/setter for the map, that now handles locking internally. For the other basic types I made the type directly exported, I think we can always change it later.